src/StoragelessSession/Session/Data.php

```diff
@@ -25,6 +25,28 @@ private function __construct(array $data, array $metadata)
         $this->metadata = $metadata;
     }
 
+    public static function fromDecodedTokenData(\stdClass $data)
+    {
+        return self::fromTokenData(self::convertStdClassToUsableStuff($data), []);
+    }
+
+    private static function convertStdClassToUsableStuff(\stdClass $shit)
```

Comment:
Maybe @lcobucci could update the jwt library to work with arrays? (I can do a PR if you agree)